src/sys11/systemConfig.py

```diff
@@ -0,0 +1,2 @@
+vectorSystem = "sys11"
+updatesURL = "http://software.warpedpinball.com/vector/sys11/latest.json"
```

Comment:
This is how we change the update URL based on system. Since this is in the sys11 folder this file will only be present on sys11 builds. there's a similar file for wpc and em. 

Importantly there's a similar one in common as well, but when you import it it will immediately raise an exception with some info. That should mean that if we forget this file in a future build we will be able to quickly know what's wrong.